spirv_msl.hpp

```diff
@@ -73,11 +74,24 @@ static const uint32_t kPushConstBinding = 0;
 class CompilerMSL : public CompilerGLSL
 {
 public:
+#define MAKE_MSL_VERSION(major, minor, patch) (((major)*10000) + ((minor)*100) + (patch))
```

Comment:
Please make this a public static function. This macro does not have a namespace which could conflict with other modules.

Reply:
I've made it a static function now.